app/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools">
 
+    <uses-permission android:name="android.permission.POST_NOTIFICATIONS" />

```

Comment:
Are we posting notifications?

Reply:
Unfortunately we need to enable notifications to make Glide work.